src/core/services/api/v1/order.service.ts

```diff
@@ -0,0 +1,183 @@
+import { client } from "@/core/services/client.service"
+import IUserRoles from "@/core/interfaces/user-roles.interface";
+import IGetAllOrdersResponse from "@/core/interfaces/order/responses/get-all-orders.interface";
+import IGetOrderByIdResponse from "@/core/interfaces/order/responses/get-order-by-id.interface";
+import IGetAllOrdersByUserIdResponse from "@/core/interfaces/order/responses/get-all-orders-by-user-id.interface";
+import ICreateOrderRequest from "@/core/interfaces/order/requests/create-order.interface";
+import IUpdateOrderRequest from "@/core/interfaces/order/requests/update-order.interface";
+
+
+const getAdminAll = async () => {
+  return await client({
+    data: '',
```

Comment:
Maybe make the data optional? so that yo don't have to declare an empty string here.

Reply:
done